### PR TITLE
Fix zip internal path issue on Windows

### PR DESCRIPTION
**Pull request by KrisJanssen**
This PR exists to fix path issues on windows when checking files inside JAR. #36

### CODE REVIEW

File: log4j-finder.py [hunk 1]
```diff
@@ -296,8 +296,8 @@ def main():
     start_time = time.monotonic()
     hostname = magenta(HOSTNAME)
 
-    if not args.no_banner and not args.quiet:
-        print(FIGLET)
+    #if not args.no_banner and not args.quiet:
```
Comment (yunzheng) [inline log4j-finder.py:299]: Please uncomment this part so it stays untouched

File: log4j-finder.py [hunk 2]
```diff
@@ -325,11 +325,9 @@ def main():
...
                             if zpath.name.lower().endswith("JndiManager.class".lower()):
-                                lookup_path = str(
-                                    zpath.parent.parent / "lookup/JndiLookup.class"
-                                )
+                                lookup_path = zpath.parent.parent / "lookup/JndiLookup.class"
                                 try:
-                                    has_lookup = zfile.open(lookup_path)
+                                    has_lookup = zfile.open(lookup_path.as_posix())
                                 except KeyError:
                                     has_lookup = False
```
Comment (yunzheng) [inline log4j-finder.py:332]: Can you set change this to “pass”. See my last commit.

**Comment by yunzheng** [comment]
Looks good! left some inline comments

File: log4j-finder.py (hunk 2 above)
Comment (KrisJanssen) [reply log4j-finder.py:332]: I believe you can just merge this? The 'as_posix()' call is still appropriate: inside zips, paths are always following posix convention.

Comment (yunzheng) [reply log4j-finder.py:332]: Sorry I was on mobile, I meant to highlight `has_look = False` in the KeyError exception handler, but that didn't seem to work. The `has_lookup = False` should just become `pass` as it is already have a default value set above the exception handler as intended.

File: log4j-finder.py (hunk 1 above)
Comment (yunzheng) [reply log4j-finder.py:299]: This was set to resolved, but I didn't see any updates.

**Comment by yunzheng** [comment]
Updated some inline comments.

File: log4j-finder.py (hunk 2 above)
Comment (yunzheng) [reply log4j-finder.py:332]: Oh nevermind, this should be set explicitly to False... i got confused.

File: log4j-finder.py (hunk 1 above)
Comment (KrisJanssen) [reply log4j-finder.py]: I amended my commit to revert the commenting out...

**Comment by yunzheng** [comment]
Looks good now, I noticed I could also edit the commit and tried it, sorry if I overwrote something in your branch. But the current state looks good. Thanks for the fix, merging!

**Comment by yunzheng** [comment]
This merge fixes #35

**Pull request merged**